feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetFavoriteSessionsFragment.kt

```diff
@@ -152,7 +151,7 @@ class BottomSheetFavoriteSessionsFragment : DaggerFragment() {
         super.onDestroyView()
     }
 
-    private fun applyTitleText() {
+    private fun applyTitleText(groupAdapter: GroupAdapter<*>) {
```

Comment:
We can use a computed property like below:

```
val groupAdapter get() = binding. sessionsRecycler.adapter as GroupAdapter<ViewHolder<*>>()
```

In addition, if this method doesn't require the concrete class, i.e. GroupAdapter, then `binding. sessionsRecycler.adapter?.itemCount ?: 0` is enough.

Reply:
Thank you for your good suggestion! 🆒 
Changed to use computed property. 2511776

Unfortunately, this method uses `GroupAdapter#getItem` 😇